run_testing_worker.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Google LLC.
```

Comment:
I think we already make changes in L23-24?

```python
#print("Starting tidb-server with logging to /tidb.log")
#subprocess.Popen(["/tidb-server", "-log-file", "/tidb.log"])
```

Reply:
You can take a look at https://github.com/tikv/client-go/blob/d535b62b62f97dc6513e0c41fefe009c6120807d/kv/key.go#L1-L31

Reply:
Done